src/main/java/com/hackathon03/codeguru/utils/InsecureCookie.java

```diff
@@ -0,0 +1,24 @@
+package com.hackathon03.codeguru.utils;
+
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+import java.net.URI;
+
+public class InsecureCookie {
+
+    public static void cookieInsecureByDefaultNoncompliant(HttpServletResponse response) {
+        Cookie cookie = new Cookie("name", "value");
+        // Noncompliant: by default, the Cookie is not secure and not httpOnly.
+        response.addCookie(cookie);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To increase the security of your code, consider setting the `httponly` attribute of cookies to `True` and `SameSite` attribute to `strict`. Setting the attribute values to `True` and `strict` prevents client-side scripts from reading the contents of the cookie.

[SameSite cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite) and [Learn more](https://cwe.mitre.org/data/definitions/1004.html)